kolibri/plugins/style_guide/assets/src/views/shell/component-docs.vue

```diff
@@ -81,7 +81,7 @@
 <script>
 
   import escodegen from 'escodegen';
-  import CamelCase from 'lodash/camelCase';
```

Comment:
I wonder if this import statement was originally deliberately ironic? (CamelCase imported not in camelCase).